lib/ret_web/controllers/page_controller.ex

```diff
@@ -139,17 +142,55 @@ defmodule RetWeb.PageController do
   end
 
   def render_index(conn) do
-    index_meta_tags = Phoenix.View.render_to_string(RetWeb.PageView, "index-meta.html", [])
+    app_config =
+      if module_config(:skip_cache) do
+        Ret.AppConfig.get_config()
+      else
+        {:ok, app_config} = Cachex.get(:app_config, :app_config)
+        app_config
+      end
+
+    app_config_json = app_config |> Poison.encode!()
+    app_config_script = "window.APP_CONFIG = JSON.parse('#{app_config_json |> String.replace("'", "\\'")}')"
```

Comment:
`window.APP = {}; window.APP.config = ... ` maybe?

Reply:
This gets put into your `configs` object anyway, so I don't think the intermediate name matters so much.